src/restaurant_inspections_api/constants.clj

```diff
@@ -0,0 +1,8 @@
+(ns restaurant-inspections-api.constants
+  (:require [restaurant-inspections-api.environment :as env]))
+
+(def ^:const port (env/get-env-port))
```

Comment:
Ugh. this might not work. might be defined at compile time instead of at runtime? The idea was to not log these values unser environment.clj every time we built, tested, and ran...